src/Microsoft.ML.Data/Model/ModelOperationsCatalog.cs

```diff
@@ -50,14 +50,49 @@ public void Save<TSource>(IDataLoader<TSource> model, Stream stream)
         }
 
         /// <summary>
-        /// Save a transformer model and the loader used to create its input data to the stream.
+        /// Save the model to the file.
+        /// </summary>
+        /// <param name="model">The trained model to be saved.</param>
+        /// <param name="filePath">Path where model should be saved.</param>
+        public void Save<TSource>(IDataLoader<TSource> model, string filePath)
+        {
+            using (var stream = File.Create(filePath))
+                Save(model, stream);
+        }
+
+        /// <summary>
+        /// Save the model to the file.
+        /// </summary>
+        /// <param name="loader">The loader that was used to create data to train the model</param>
+        /// <param name="model">The trained model to be saved</param>
+        /// <param name="filePath">Path where model should be saved.</param>
+        public void Save<TSource>(IDataLoader<TSource> loader, ITransformer model, string filePath)
+        {
+            using (var stream = File.Create(filePath))
+                Save(loader, model, stream);
+        }
+
+        /// <summary>
+        /// Load the model from the stream.
         /// </summary>
         /// <param name="loader">The loader that was used to create data to train the model</param>
         /// <param name="model">The trained model to be saved</param>
         /// <param name="stream">A writeable, seekable stream to save to.</param>
         public void Save<TSource>(IDataLoader<TSource> loader, ITransformer model, Stream stream) =>
             Save(new CompositeDataLoader<TSource, ITransformer>(loader, new TransformerChain<ITransformer>(model)), stream);
 
+        /// <summary>
+        /// Save the model to the file.
```

Comment:
>Save the model to the file. [](start = 12, length = 27)

Save a transformer model and the loader used to create its input data to the stream.